SPECS/libnvidia-container/libnvidia-container.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
- "Signatures": {
-  "libnvidia-container-1.17.3.tar.gz": "9c28e729f1677b6ff9edb3f4dccff4f9827ffe13c04fbcffa41aabacbc0fdd54",
-  "nvidia-modprobe-550.54.14.tar.gz": "5687b0dfa6087dd480ae91e91ff1dca975794e35a2edcf9ec08d8f9cb98ef905"
- }
-}
\ No newline at end of file
+  "Signatures": {
+    "nvidia-modprobe-550.54.14.tar.gz": "5687b0dfa6087dd480ae91e91ff1dca975794e35a2edcf9ec08d8f9cb98ef905",
```

Comment:
Can we upgrade this vendor source tar version since our GPU driver version has been upgraded to v560.35.03? This should have been changed when the CUDA driver version was bumped to 560.35.03 previously. The existing implementation should not cause a regression given that our pre-release containerized GPU workload testing passed successfully, but still would be good to keep the version in-sync to be completely robust.

Reply:
@miz060 discovered that the upstream source is still pinning to the older nvidia modprobe version of v550.54.14. Discussed offline and the current plan is to keep the version on the older 550.54.14 to align with the upstream change. Will bring this issue up with NVIDIA upstream to fully corroborate on future version upgrade strategy of this vendor source.